crates/accelerate/src/sabre_layout.rs

```diff
@@ -97,7 +96,7 @@ pub fn sabre_layout_and_routing(
     };
     (
         res.0,
-        res.1,
+        PyArray::from_vec(py, res.1).into(),
```

Comment:
Hah I didn't even know this method or `from_iter()` existed. Looking at the rust-numpy source it looks like it just calls `into_pyarray()` internally. I wonder if this just my knowledge of the library is several years out of date and these didn't exist when I started using it. It's a little bit more ergonomic looking than the `.into_pyarray()` I used everywhere else to do a no copy numpy array creation.